lerna-package-lock.json

```diff
@@ -2004,13 +2004,13 @@
 			}
 		},
 		"@fluidframework/container-definitions": {
-			"version": "0.43.0-45414",
-			"resolved": "https://registry.npmjs.org/@fluidframework/container-definitions/-/container-definitions-0.43.0-45414.tgz",
-			"integrity": "sha512-0JEO3YdjpfTOaC7stCcaDg82m5eTTcKMBbag4BwKssmDPmda8vOn9MjsphaMSzXw5WYM6i24OBC43mTiyFN3Ng==",
+			"version": "0.43.0",
```

Comment:
@markfields this is what's causing the problem. between prelease 0.43.0-45414 and release 0.43.0 some change came in that is just now being consumed.

Reply:
Ok that makes sense